MuxExoPlayer/build.gradle

```diff
@@ -28,7 +30,7 @@ android {
         minSdkVersion project.ext.minSdkVersion
         targetSdkVersion project.ext.targetSdkVersion
         versionCode 26
-        versionName "2.4.15"
+        versionName project.ext.versionName
```

Comment:
Just one quick question - this is set correctly elsewhere as 2.4.15, right?

Reply:
It'll base the version name on the git commit message for PRs merged to master. Nowhere is this now set to 2.4.15. If the commit message starts with a vX.X.X pattern then it'll assume you want to make a public release build with that version

In the event that it's not a merge to master (i.e. it's almost anything else that would trigger a build) you will still get a build on our (to be) private maven repo name "dev-[branch]-[hash]" for testing purposes.

Reply:
oh even better, works for me!